factorial/fact.cpp

```diff
@@ -0,0 +1,36 @@
+#include<iostream>
+int fun(int n)
+{
+if(n==0)
+    {
+        return 1;
+    }
+    else
+    {
+            n=n*fun(n-1);
+    }
+}
+int main()
+{
+    int n;
+    char c;
+    std::cout<<"Enter a positive Integer to find Factorial: ";
+    std::cin>>n;
+    std::cout<<"\n";
+    if(n>=0 && n!=c)
+    {
+     n=fun(n);
```

Comment:
What will happen if user wants to calculate factorial of any number greater than 13? Will your code work in that case?